src/model/ai/evaluation/PawnDistanceToKingedEvaluator.java

```diff
@@ -0,0 +1,54 @@
+package model.ai.evaluation;
+
+import model.Board;
+import model.PieceColor;
+import model.PieceInterface;
+import model.Square;
+
+public class PawnDistanceToKingedEvaluator implements BoardEvaluatorInterface {
+
+    private static BoardEvaluatorInterface instance = null;
+
+    public static BoardEvaluatorInterface getInstance() {
+        if (instance == null) {
+            return new PawnDistanceToKingedEvaluator();
```

Comment:
You want to make this line be `instance = new PawnDistanceToKingedEvaluator();` otherwise your singleton will never maintain the newly created instance of your object, and thus isn't very singleton-y. :)